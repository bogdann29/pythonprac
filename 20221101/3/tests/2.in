f, g = Grange(1, 1, 1), Grange(10, 2, 21)
o = Grange(100, 10, 200)
print(f"{f=}, {g=}")
print(f"{list(o)}({not(o)})")
print(f"{len(f)=}, {len(g)=}")
print(f"{len(list(f))=}, {len(list(g))=}, {len(list(o))=}")
print(f"{g[0]=}, {o[11]=}")
a = g[3:32767]
print(f"{a=}, {a[5]=}")