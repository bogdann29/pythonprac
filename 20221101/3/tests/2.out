f=grange(1, 1, 1), g=grange(10, 2, 21)
[100](False)
len(f)=0, len(g)=2
len(list(f))=0, len(list(g))=2, len(list(o))=1
g[0]=10, o[11]=10000000000000
a=grange(3, 2, 32767), a[5]=96
